### PR TITLE
Implement initial settings screen for config

### PR DESCRIPTION
**Pull request by mxwnk**

### CODE REVIEW

File: frontend-mobile/lib/view/coffee_call_screen.dart [hunk 1]
```diff
@@ -18,6 +19,19 @@ class CoffeeCallScreen extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Coffee Caller'),
+        actions: [
+          TextButton.icon(
+            style: TextButton.styleFrom(foregroundColor: Colors.white),
+            onPressed: () {
+              Navigator.push(
+                context,
+                MaterialPageRoute(builder: (context) => const SettingsScreen()),
```
Comment (mxwnk) [inline frontend-mobile/lib/view/coffee_call_screen.dart:25]: @t-animal do you have any opinion on named routes?

Comment (t-animal) [reply frontend-mobile/lib/view/coffee_call_screen.dart:25]: In the past I've used generated named routes, but that was before the introduction of the new declarative `Router` api. Generated routes and the router api would be overkill at this point, but we should keep in mind that we may have to switch to it when we introduce more pages.

File: frontend-mobile/lib/view/settings_screen.dart [hunk 2]
```diff
@@ -9,9 +9,6 @@ class SettingsScreen extends StatelessWidget {
         appBar: AppBar(
           title: const Text('Settings'),
         ),
-        body: const Center(
-            child: Text('Hello Settings')
-        )
-    );
+        body: const Center(child: Text('Hello Settings')));
```
Comment (t-animal) [inline frontend-mobile/lib/view/settings_screen.dart]: If you want to keep indentation, add a comma between the braces. It's annoying, I know.

**Review by t-animal — APPROVED** [review]

File: frontend-mobile/lib/view/settings_screen.dart (hunk 2 above)
Comment (mxwnk) [reply frontend-mobile/lib/view/settings_screen.dart]: Thanks for the hint! Done :)

**Pull request merged**